source/server/index.js

```diff
@@ -1,3 +1,8 @@
-import ServerApp from '/source/imports/application/server-application.js'
+import ServerApp from '../imports/application/server-application.js';
 
-ServerApp.start();
\ No newline at end of file
+Meteor.startup(function() {
+
+  if (Meteor.isServer) {
+    ServerApp.start();
```

Comment:
I'd suggest instantiating the ServerApp instance here, keeping the class file purely a definition